.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,29 +7,35 @@ assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is
+### Describe the bug
+<!-- A clear and concise description of what the bug is -->
 
-**To Reproduce**
-Steps to reproduce the behavior (attach screenshots if applicable):
+### To Reproduce
+<!--Steps to reproduce the behavior (attach screenshots if applicable):
+    e.g.
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
-4. See error
+4. See error -->
 
-**What did you expect to happen?**
-A clear and concise description of what you expected to happen.
+### What did you expect to happen?
+<!--A clear and concise description of what you expected to happen. -->
 
-**Browser Environment (please complete the following information):**
-- OS: [e.g. Plan9, HURD, Oberon, etc]
-- Browser [e.g. chrome, firefox] (note that only these two browsers are supported)
-- Version [e.g. 22]
+### Your Environment
+<!--[e.g. MacOS Catalina, Windows 10 -->
+- Your OS: 
+<!-- Only Chrome and Firefox are supported. -->
+- Browser: 
 
-**Server Environment (if known):**
-_Note: We only support the most recent release of LORIS._
-- LORIS Version: [e.g. 22.0.0]
-- Linux distribution and Version: [e.g. Ubuntu 16.04, CentOS 7]
-- MySQL/MariaDB Version: [e.g. MySQL 5.7, MariaDB 10.3]
+### Server Environment (if known):
+<!-- We only support the most recent release of LORIS.
```

Comment:
delete this line and the next (somehow multi-line suggest wouldn't work here, sorry)